### PR TITLE
Viewer script

### PR DESCRIPTION
**Pull request by jburel**
Paris 2015 feedback (Jim at JAX)
* add option to run script from the image viewer

To test:
 * Open an image in full view
 * Check that a button to run script is visible in viewer toolbar
 * Click on it,
 * Check that the script menu pops up

### CODE REVIEW

File: components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/view/ToolBar.java [hunk 1]
```diff
@@ -42,9 +44,11 @@
 import javax.swing.JToggleButton;
 import javax.swing.JToolBar;
 
+
+
 //Third-party libraries
 import org.jdesktop.swingx.JXBusyLabel;
-
+import org.openmicroscopy.shoola.agents.events.iviewer.ScriptDisplay;
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/view/ToolBar.java]: application-internal

File: components/insight/SRC/org/openmicroscopy/shoola/agents/treeviewer/ScriptsLoader.java [hunk 2]
```diff
@@ -26,11 +26,19 @@
...
 
 //Third-party libraries
 
+
+
+
+import javax.swing.JButton;
+
+
+import org.openmicroscopy.shoola.agents.events.iviewer.ScriptDisplay;
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/agents/treeviewer/ScriptsLoader.java]: application-internal, and lots of whitespace above

**Comment by mtbc** [comment]
Works very well, tested with various images / scripts.

Two weird things in trying out channel offsets script:

* Insight lets me type a negative offset, Web doesn't.
* On a big image, the channel offsets script produces an error that Insight seems to swallow silently.

These are probably unrelated to this PR, though; I could open tickets if need be.

**Comment by jburel** [comment]
@mtbc: not related to the PRs. no script modified Please open a ticket.

**Comment by mtbc** [comment]
Thank you. :+1:

**Comment by jburel** [comment]
@mtbc: cleaned up import section

**Comment by jburel** [comment]
@gusferguson: this change will probably require an update in the help i.e. ability to run script from viewer

**Comment by gusferguson** [comment]
@jburel - thanks I have made a note - aiming for 5.1.3?

**Comment by jburel** [comment]
@gusferguson: yes

**Comment by mtbc** [comment]
Good to merge.

**Comment by jburel** [comment]
Merging this PR so the conflict can be fixed.

**Comment by joshmoore** [comment]
I assume this gets screenshots, @gusferguson ?

**Comment by gusferguson** [comment]
@joshmoore - updating them as we speak.

**Pull request merged**